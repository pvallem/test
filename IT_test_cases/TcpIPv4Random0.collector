bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\333\211\316{"
    src_port: 58060
    dst_port: 46473
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200016252
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200016252
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200016252
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\304GW\276"
    src_port: 51682
    dst_port: 16945
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200013710
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200013710
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200013710
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\266\232<\356"
    src_port: 45543
    dst_port: 40269
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200021065
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200021065
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200021065
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\321\347\330\035"
    src_port: 51682
    dst_port: 35527
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200030962
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200030962
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200030962
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\321\347\330\035"
    src_port: 58060
    dst_port: 41658
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200024878
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200024878
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 200
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200024878
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\212\235\362b"
    dst_address: "\233>d\350"
    src_port: 17525
    dst_port: 41389
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200023607
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200023607
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200023607
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\321\347\330\035"
    src_port: 41389
    dst_port: 42161
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200006355
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200006355
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200006355
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\333\211\316{"
    src_port: 53312
    dst_port: 40269
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200034775
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200034775
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 280
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200034775
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\333\211\316{"
    src_port: 25446
    dst_port: 17525
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200009897
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200009897
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 80
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200009897
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 45543
    dst_port: 41658
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200038588
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200038588
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200038588
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\266\232<\356"
    src_port: 58060
    dst_port: 42161
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200033504
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200033504
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200033504
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 53312
    dst_port: 16945
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 0
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000000
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\212\235\362b"
    src_port: 25446
    dst_port: 18766
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200036046
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200036046
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200036046
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\212\235\362b"
    dst_address: "\233>d\350"
    src_port: 6835
    dst_port: 51682
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200005084
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200005084
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 40
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200005084
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 16945
    dst_port: 42161
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200019794
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200019794
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 160
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200019794
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\333\211\316{"
    src_port: 45543
    dst_port: 35527
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200003813
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200003813
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200003813
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\333\211\316{"
    src_port: 16945
    dst_port: 5312
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200028691
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200028691
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200028691
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\333\211\316{"
    src_port: 51682
    dst_port: 18766
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200022336
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200022336
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 180
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200022336
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\321\347\330\035"
    src_port: 45543
    dst_port: 15522
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200012439
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200012439
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 100
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200012439
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\266\232<\356"
    src_port: 16945
    dst_port: 46473
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200002542
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200002542
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 20
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200002542
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\212\235\362b"
    dst_address: "\242f\310\323"
    src_port: 46473
    dst_port: 45543
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200029691
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200029691
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 240
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200029691
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 25446
    dst_port: 6835
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200018794
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200018794
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200018794
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\266\232<\356"
    src_port: 41389
    dst_port: 5312
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200014981
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200014981
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 120
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200014981
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\266\232<\356"
    src_port: 53312
    dst_port: 18766
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200008897
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200008897
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200008897
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\212\235\362b"
    src_port: 53312
    dst_port: 35527
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200017523
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200017523
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 140
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200017523
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\212\235\362b"
    dst_address: "\242f\310\323"
    src_port: 41658
    dst_port: 16945
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200011168
    key_type: IPV4
  }
  reverse_flow_info {
    start_time: 1388563200011168
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200011168
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\304GW\276"
    src_port: 53312
    dst_port: 15522
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200026149
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200026149
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200026149
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\266\232<\356"
    src_port: 25446
    dst_port: 16945
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200027420
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200027420
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 220
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200027420
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 16945
    dst_port: 17525
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200037317
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200037317
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 300
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200037317
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\304GW\276"
    src_port: 58060
    dst_port: 40269
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200007626
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200007626
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 60
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200007626
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 25446
    dst_port: 5312
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200001271
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200001271
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200001271
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\304GW\276"
    src_port: 41389
    dst_port: 6835
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200032233
    key_type: IPV4
  }
  flow_info {
    start_time: 1388563200032233
    ip_info {
      last_cached_ttl: 8
      qos: 0
    }
    tcp_info {
      sequence_num: 260
      null_flags: 1
      syn_fin_rst_ack_all_zero: 1
      option_len_bins {
      }
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200032233
    sensor_id: "127.0.0.1:5675"
  }
}
