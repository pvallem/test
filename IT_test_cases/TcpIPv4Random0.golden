flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 53312
    dst_port: 16945
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: IPV4
  }
  start_time: 1388563200000000
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 0
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000000
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 25446
    dst_port: 5312
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200001271
    key_type: IPV4
  }
  start_time: 1388563200001271
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200001271
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\266\232<\356"
    src_port: 16945
    dst_port: 46473
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200002542
    key_type: IPV4
  }
  start_time: 1388563200002542
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 20
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200002542
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\333\211\316{"
    src_port: 45543
    dst_port: 35527
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200003813
    key_type: IPV4
  }
  start_time: 1388563200003813
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200003813
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\212\235\362b"
    src_port: 51682
    dst_port: 6835
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200005084
    key_type: IPV4
  }
  start_time: 1388563200005084
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 40
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200005084
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\321\347\330\035"
    src_port: 41389
    dst_port: 42161
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200006355
    key_type: IPV4
  }
  start_time: 1388563200006355
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200006355
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\304GW\276"
    src_port: 58060
    dst_port: 40269
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200007626
    key_type: IPV4
  }
  start_time: 1388563200007626
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 60
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200007626
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\266\232<\356"
    src_port: 53312
    dst_port: 18766
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200008897
    key_type: IPV4
  }
  start_time: 1388563200008897
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200008897
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\333\211\316{"
    src_port: 25446
    dst_port: 17525
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200009897
    key_type: IPV4
  }
  start_time: 1388563200009897
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 80
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200009897
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\212\235\362b"
    src_port: 16945
    dst_port: 41658
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200011168
    key_type: IPV4
  }
  start_time: 1388563200011168
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200011168
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\321\347\330\035"
    src_port: 45543
    dst_port: 15522
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200012439
    key_type: IPV4
  }
  start_time: 1388563200012439
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 100
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200012439
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\304GW\276"
    src_port: 51682
    dst_port: 16945
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200013710
    key_type: IPV4
  }
  start_time: 1388563200013710
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200013710
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\266\232<\356"
    src_port: 41389
    dst_port: 5312
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200014981
    key_type: IPV4
  }
  start_time: 1388563200014981
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 120
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200014981
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\333\211\316{"
    src_port: 58060
    dst_port: 46473
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200016252
    key_type: IPV4
  }
  start_time: 1388563200016252
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200016252
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\212\235\362b"
    src_port: 53312
    dst_port: 35527
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200017523
    key_type: IPV4
  }
  start_time: 1388563200017523
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 140
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200017523
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 25446
    dst_port: 6835
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200018794
    key_type: IPV4
  }
  start_time: 1388563200018794
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200018794
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 16945
    dst_port: 42161
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200019794
    key_type: IPV4
  }
  start_time: 1388563200019794
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 160
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200019794
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\266\232<\356"
    src_port: 45543
    dst_port: 40269
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200021065
    key_type: IPV4
  }
  start_time: 1388563200021065
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200021065
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\333\211\316{"
    src_port: 51682
    dst_port: 18766
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200022336
    key_type: IPV4
  }
  start_time: 1388563200022336
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 180
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200022336
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\212\235\362b"
    src_port: 41389
    dst_port: 17525
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200023607
    key_type: IPV4
  }
  start_time: 1388563200023607
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200023607
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\321\347\330\035"
    src_port: 58060
    dst_port: 41658
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200024878
    key_type: IPV4
  }
  start_time: 1388563200024878
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 200
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200024878
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\304GW\276"
    src_port: 53312
    dst_port: 15522
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200026149
    key_type: IPV4
  }
  start_time: 1388563200026149
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200026149
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\266\232<\356"
    src_port: 25446
    dst_port: 16945
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200027420
    key_type: IPV4
  }
  start_time: 1388563200027420
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 220
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200027420
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\333\211\316{"
    src_port: 16945
    dst_port: 5312
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200028691
    key_type: IPV4
  }
  start_time: 1388563200028691
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200028691
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\212\235\362b"
    src_port: 45543
    dst_port: 46473
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200029691
    key_type: IPV4
  }
  start_time: 1388563200029691
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 240
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200029691
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\321\347\330\035"
    src_port: 51682
    dst_port: 35527
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200030962
    key_type: IPV4
  }
  start_time: 1388563200030962
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200030962
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\304GW\276"
    src_port: 41389
    dst_port: 6835
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200032233
    key_type: IPV4
  }
  start_time: 1388563200032233
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 260
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200032233
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\266\232<\356"
    src_port: 58060
    dst_port: 42161
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200033504
    key_type: IPV4
  }
  start_time: 1388563200033504
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200033504
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\333\211\316{"
    src_port: 53312
    dst_port: 40269
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200034775
    key_type: IPV4
  }
  start_time: 1388563200034775
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 280
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200034775
}
flow_info {
  key {
    src_address: "\016*\365B"
    dst_address: "\212\235\362b"
    src_port: 25446
    dst_port: 18766
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200036046
    key_type: IPV4
  }
  start_time: 1388563200036046
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200036046
}
flow_info {
  key {
    src_address: "\242f\310\323"
    dst_address: "\321\347\330\035"
    src_port: 16945
    dst_port: 17525
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200037317
    key_type: IPV4
  }
  start_time: 1388563200037317
  ip_info {
    last_cached_ttl: 8
    qos: 0
  }
  tcp_info {
    sequence_num: 300
    null_flags: 1
    syn_fin_rst_ack_all_zero: 1
    option_len_bins {
    }
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200037317
}
flow_info {
  key {
    src_address: "\233>d\350"
    dst_address: "\304GW\276"
    src_port: 45543
    dst_port: 41658
    proto: 6
    tenant_id: 20
    flow_start_time: 1388563200038588
    key_type: IPV4
  }
  start_time: 1388563200038588
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200038588
}
